apps/molgenis-components/src/components/tables/RefSideModal.vue

```diff
@@ -1,15 +1,23 @@
 <template>
-  <SideModal :label="label" :isVisible="true" @onClose="emit('onClose')">
+  <SideModal
+    :label="localColumnName"
+    :isVisible="true"
+    @onClose="emit('onClose')"
+  >
     <div v-if="loading">
       <Spinner />
     </div>
     <div v-else>
       <MessageError v-if="errorMessage">{{ errorMessage }}</MessageError>
+      <div v-if="!queryResults.length">
+        The selected reference was not found in the database
```

Comment:
Can we show this missing reference? Because in practice this shouldn't happen right?

Reply:
Hmm gonna leave this for now. Should be possible, but needs some thinking.